.nf-core.yml

```diff
@@ -8,3 +8,4 @@ bump-versions:
   rseqc/inferexperiment: False
   rseqc/innerdistance: False
   sortmerna: False
+org_path: nf-core
```

Comment:
What is this new line?

Reply:
Thank you for pointing this out.  I don't think it should be included in the commit.  That line is created when running `nf-core modules create-test-yml ashlar`, so you don't have to enter the org_path each time.  This may be different for other users though, so that's why it isn't in the master branch.  I've removed it now and the tests run fine as expected.